lib/pdf/core/pdf_object.rb

```diff
@@ -36,6 +36,8 @@ def string_to_hex(str)
 
     ESCAPED_NAME_CHARACTERS = (1..32).to_a + [35, 40, 41, 47, 60, 62] + (127..255).to_a
 
+    STRING_ESCAPE_MAP = { '(' => '\(', ')' => '\)', '\\' => '\\\\', "\r" => '\\r' }.freeze
```

Comment:
Does the backslash in a single-quoted string need to be escaped?
```ruby
'\r'.bytes # => [92, 114]
'\\r'.bytes # => [92, 114]
```
It's not incorrect. Just a little bit more confusing that it neds to be.

Reply:
Honestly I'm never sure what to escape and what not with single-quoted strings, so I only use them when there are no backslashes :shrug:

So if `'\r'` also works, fine by me :+1: